observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -290,6 +293,153 @@ include::snippets/technology-preview.adoc[]
+[discrete]
+[id="log-group-name_{context}"]
+=== Log group name
+
+The `log_group_name` parameter allows you to customize the log group name and supports the default `+/metrics/default+` value or the following placeholders:
+
+`+/aws/metrics/{ClusterName}+`:: This placeholder is used to search for the `ClusterName` or `+aws.ecs.cluster.name+` resource attribute in the metrics data and replace it with the actual cluster name.
+
+`+{NodeName}+`:: This placeholder is used to search for the `NodeName` or `+k8s.node.name+` resource attribute.
+
+`+{TaskId}+`:: This placeholder is used to search for the `TaskId` or `+aws.ecs.task.id+` resource attribute.
+
+If no resource attribute is found in the resource attribute map, the placeholder is replaced by the `undefined` value.
+
+[discrete]
+[id="log-stream-name_{context}"]
+=== Log stream name
+
+The `log_stream_name` parameter allows you to customize the log stream name and supports the default `+otel-stream+` value or the following placeholders:
+
+`+{ClusterName}+`:: This placeholder is used to search for the `+ClusterName+` or `+aws.ecs.cluster.name+` resource attribute.
+
+`+{ContainerInstanceId}+`:: This placeholder is used to search for the `+ContainerInstanceId+` or `+aws.ecs.container.instance.id+` resource attribute. This resource attribute is valid only for the AWS ECS EC2 launch type.
+
+`+{NodeName}+`:: This placeholder is used to search for the `+NodeName+` or `+k8s.node.name+` resource attribute.
+
+`+{TaskDefinitionFamily}+`:: This placeholder is used to search for the `+TaskDefinitionFamily+` or `+aws.ecs.task.family+` resource attribute.
+
+`+{TaskId}+`:: This placeholder is used to search for the `TaskId` or `+aws.ecs.task.id+` resource attribute in the metrics data and replace it with the actual task ID.
+
+If no resource attribute is found in the resource attribute map, the placeholder is replaced by the `undefined` value.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html[Specification: Embedded metric format] (Amazon CloudWatch User Guide)
+* link:https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html[PutLogEvents] (Amazon CloudWatch Logs API Reference)
+* link:https://docs.aws.amazon.com/general/latest/gr/cwl_region.html[Service endpoints] (AWS General Reference)
+
+[id="aws-xray-exporter_{context}"]
+== AWS X-Ray Exporter
+
+The AWS X-Ray Exporter converts OpenTelemetry spans to AWS X-Ray Segment Documents and then sends them directly to the AWS X-Ray service by using the `PutTraceSegments` API and signing the requests by the AWS SDK for Go using the default credential provider chain.
+
+:FeatureName: The AWS X-Ray Exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled AWS X-Ray Exporter
+[source,yaml]
+----
+# ...
+  config:
+    exporters:
+      awsxray:
```

Comment:
Endpoint is an important optional parameter that customers would override. https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsxrayexporter#exporter-configuration

Reply:
@IshwarKanse, please see my addition lines [426](https://github.com/openshift/openshift-docs/pull/88479/files#diff-4182f1a314619ad34871720858dd9e37c4c0797d1eb66028dfa410022a2eaa3cR426) and [435](https://github.com/openshift/openshift-docs/pull/88479/files#diff-4182f1a314619ad34871720858dd9e37c4c0797d1eb66028dfa410022a2eaa3cR435). Is this okay?